doc-content/enterprise-only/springboot/spring-boot-jms-audit-proc.adoc

```diff
@@ -0,0 +1,102 @@
++
+[source, xml]
+----
+<dependency>
+  <groupId>org.messaginghub</groupId>
+  <artifactId>pooled-jms</artifactId>
+</dependency>
+----
+. To configure {KIE_SERVER} audit replication to use queues, complete the following tasks:
++
+.. Add the following lines to your Spring Boot application's `application.properties` file where `<JMS_BROKER>` is the JMS broker that you want to connect to, `<PORT>` is the port that the broker listens on, and  `<USERNAME>` and `<PASSWORD` are the login credentials for the broker:
++
+[source]
+----
+kieserver.audit-replication.producer=true
+kieserver.audit-replication.queue=audit-queue
+<JMS_BROKER>.remote-url=amqp://localhost:<PORT>
+<JMS_BROKER>.username=<USERNAME>
+<JMS_BROKER>.password=<PASSWORD>
+<JMS_BROKER>.pool.enabled=true
+----
+
+.. Add the following lines to the `application.properties` file of the service that will consume the message broker data, where `<JMS_BROKER>` is the JMS broker that your consumer will connect to, `<PORT>` is the port that your message broker listens on and  `<USERNAME>` and `<PASSWORD>` are the login credentials for the message broker:
++
+[source]
+----
+kieserver.audit-replication.consumer=true
+kieserver.audit-replication.queue=audit-queue
+<JMS_BROKER>.remote-url=amqp://localhost:<PORT>
+<JMS_BROKER>.username=<USERNAME>
+<JMS_BROKER>.password=<PASSWORD>
+<JMS_BROKER>.pool.enabled=true
+----
+. To configure {KIE_SERVER} audit replication to use topics, complete the following tasks:
++
+.. Add the following lines to your Spring Boot application's `application.properties` file where `<JMS_BROKER>` is the JMS broker that you want to connect to, `<PORT>` is the port that your message broker listens on and  `<USERNAME>` and `<PASSWORD` are the login credentials for the message broker:
++
+[source]
+----
+kieserver.audit-replication.producer=true
+kieserver.audit-replication.topic=audit-topic
+spring.jms.pub-sub-domain=true
+<JMS_BROKER>.remote-url=amqp://localhost:<PORT>
+<JMS_BROKER>.username=<USERNAME>
+<JMS_BROKER>.password=<PASSWORD>
+<JMS_BROKER>.pool.enabled=true
+----
+.. Add the following lines to the `application.properties` file of the service that will consume the message broker data, where `<JMS_BROKER>` is the JMS broker that your consumer will connect to, `<PORT>` is the port that your message broker listens on and  `<USERNAME>` and `<PASSWORD>` are the login credentials for the message broker:
++
+[source]
+----
+kieserver.audit-replication.consumer=true
+kieserver.audit-replication.topic=audit-topic::jbpm
+kieserver.audit-replication.topic.subscriber=jbpm
+spring.jms.pub-sub-domain=true
+<JMS_BROKER>.remote-url=amqp://localhost:<PORT>
+<JMS_BROKER>.username=<USERNAME>
+<JMS_BROKER>.password=<PASSWORD>
+<JMS_BROKER>.pool.enabled=true
+<JMS_BROKER>.clientId=jbpm
+----
+. To configure your application to be read-only so that users can query the replicated audit data but not change anything, add the following line to the `application.properties` file:
```

Comment:
The user can query the data either the server is in read only mode or not. This is a confused statement. It would be something like: If your want your kie server with the replicated data to be read only you need to setup this property.